src/ReactWrapper.jsx

```diff
@@ -107,6 +108,26 @@ export default class ReactWrapper {
   }
 
   /**
+   * Makes a wrapper iterable, which is useful when using destructive
+   * assignment with `find()`
+   * @example
+   * const wrapper = shallow(<MyComponent />)
+   * const [fistLink, secondLink] = wrapper.find('a')
+   * @returns {Object}
+   */
+  [ITERATOR_SYMBOL]() {
+    let index = 0;
```

Comment:
Yes, let's please only provide the function when the environment supports iterables.